content/articles/domain-list.md

```diff
@@ -10,97 +10,97 @@ categories:
 
-![](/files/domain-list-domain-name-arrow.png)
+![screenshot: Showing link to domain names in the navigation](/files/domain-list-domain-name-arrow.png)
 
-The account you're viewing is bolded so you can verify you're in the correct account. 
+The account you're viewing is bolded so you can verify you're in the correct account.
 
 ## List view
 
-The page displays the first 50 domains in your account. 
+The page displays the first 50 domains in your account.
 
 If you have more than 50, use the pagination under the domain list to view next and previous pages, or jump to a specific page.
 
 Each domain has an overview with the following information:
 
-![](/files/domain-list-headers.png)
+![screenshot: Showing table headers for the domain list table](/files/domain-list-headers.png)
 
- ### Domain Name 
- A clickable link to the domain-specific home page to view, edit and manage the DNS and Domain Management. 
+ ### Domain Name
+ A clickable link to the domain-specific home page to view, edit and manage the DNS and Domain Management.
 
-### Domain Registration 
+### Domain Registration
 The state of the domain in DNSimple.
  - **Transfer Domain** - indicates the domain is not registered with DNSimple or an Integrated Provider. Click the link attached to begin the transfer process.
  - **Registered** - indicates the domain is registered with DNSimple or an Integrated Provider. Clicking Registered will take you directly to the domain's registration hub.
    - If the domain is registered with DNSimple, it will display the expiration date, and auto-renew (if enabled). If the domain is not set to auto-renew, it will not be included in the quick view, and can be turned on by following [these steps](/articles/domain-auto-renewal/).
    - If the domain is registered with an Integrated Provider, that provider is also displayed.
 
-![](/files/domain-list-godaddy-autorenew.png)
+![screenshot: Showing a domain that is registered with GoDaddy](/files/domain-list-godaddy-autorenew.png)
 
 ### DNS Zones
 
 This indicates whether you have activated the zones and if the domain is resolving. Click the link attached to the zone status to navigate to the domain's DNS management page.
 
 If it's active and resolving with DNSimple, it will appear like this:
 
-![](/files/domain-list-active-zone.png)
+![screenshot: Showing active domain](/files/domain-list-active-zone.png)
 
 If it's active and resolving with an Integrated Provider, that provider will also be displayed.
 
-![](/files/domain-list-active-zone-route53.png)
+![screenshot: Showing domain which has activated route53 as a zone](/files/domain-list-active-zone-route53.png)
 
 If the domain does not have the zones activated within your DNSimple account, it will show as Inactive.
 
-![](/files/domain-list-inactive-zone.png)
+![screenshot: Showing domain which is inactive](/files/domain-list-inactive-zone.png)
```

Comment:
I'm wondering about the "screenshot" prefix. As this is mostly for SEO purposes, does it provide value?

Reply:
i was trying to follow the convention i had seen on some of the other screenshots. That may not effect SEO, but IMO could be good for accessibility, indicating it is a screenshot of UI?